lib/versioning/swift/index.spec.ts

```diff
@@ -86,6 +90,7 @@ describe('versioning/swift/index', () => {
   `(
     'isLessThanRange("$version", "$range") === "$expected"',
     ({ version, range, expected }) => {
+      assert(isLessThanRange);
```

Comment:
Why assert?

Reply:
At the morning, it seemed to me the most elegant way to tell compiler the optional function is really exists for this particular `VersioningApi`  instance 😅